geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/JobUtils.scala

```diff
@@ -24,7 +24,10 @@ object JobUtils extends LazyLogging {
    * @param libJars jar prefixes to load
    */
   def setLibJars(conf: Configuration, libJars: Seq[String], searchPath: Iterator[() => Seq[File]]): Unit = {
-    val paths = ClassPathUtils.findJars(libJars, searchPath).map(f => "file:///" + f.getAbsolutePath)
+    val extra = ClassPathUtils.loadClassPathFromEnv("GEOMESA_EXTRA_CLASSPATHS")
```

Comment:
will `libjars` work with non-jar files? for some reason i thought that they actually had to end with .jar